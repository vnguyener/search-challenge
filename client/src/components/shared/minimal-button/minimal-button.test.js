import React from 'react';
import renderer from 'react-test-renderer';

import MinimalButton from '.';

describe('minimal button component', () => {
  it('renders correctly', () => {
    const element = renderer.create(<MinimalButton />).toJSON();
    expect(element).toMatchSnapshot();
  });
});
