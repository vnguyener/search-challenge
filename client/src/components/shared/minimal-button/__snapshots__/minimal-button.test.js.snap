// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`minimal button component renders correctly 1`] = `
<button
  className="minimal-button"
  style={Object {}}
/>
`;
